src/signin/signinReducers.js

```diff
@@ -0,0 +1,19 @@
+const initialState = {
+  token: null,
+  userName: null,
+  isAuthenticated: false,
+  isAuthenticating: false,
+  statusMessage: null,
+};
+
+export default function (state = initialState, action) {
```

Comment:
Does this pass the linter? I thought airbnb required names for functions expressions.